src/org/ods/services/OpenShiftService.groovy

```diff
@@ -301,7 +301,7 @@ class OpenShiftService {
 
     int startBuild(String project, String name, String dir) {
         steps.sh(
-            script: "oc -n ${project} start-build ${name} --from-dir ${dir}",
+            script: "oc -n ${project} start-build ${name} --from-dir ${dir} ${logger.OCPDebugFlag}",
```

Comment:
Love the addition. As a nitpick, could we use `logger.ocDebugFlag`? I think that would better reflect what the flag is for (the `oc` binary) and avoids `OCPD` all uppercase ;)

Reply:
nitpick, thats right ... I'll change that